wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/WireMockServlet.java

```diff
@@ -5,15 +5,16 @@
 import javax.servlet.http.HttpServletResponse;
 
 import com.cognifide.aem.stubs.wiremock.WireMockApp;
+import com.cognifide.aem.stubs.wiremock.cors.CorsConfiguration;
 import com.cognifide.aem.stubs.wiremock.servlet.handler.MocksHandlerFactory;
 
 public class WireMockServlet extends HttpServlet {
 
   private final MocksHandlerFactory factory;
 
-  public WireMockServlet(String path, WireMockApp app) {
+  public WireMockServlet(String path, WireMockApp app, CorsConfiguration corsConfiguration) {
```

Comment:
what if we will have more CORS-like features in the future... then this constructor may grow too much